dev/com.ibm.ws.wsoc/src/com/ibm/ws/wsoc/outbound/HttpRequestor.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2014 IBM Corporation and others.
+ * Copyright (c) 2022 IBM Corporation and others.
```

Comment:
This is technically a new file, so I used just 2022.  All the old code is in `HttpRequestorWsoc10`